reana_client/cli/workflow.py

```diff
@@ -308,6 +308,51 @@ def workflow_start(ctx, workflow, access_token):
 def workflow_status(ctx, workflow, _filter, output_format,
                     access_token, verbose):
     """Get status of previously created workflow."""
+    def _show_progress(succeeded_jobs, total_jobs):
+        if total_jobs:
+            return '{0}/{1}'.format(succeeded_jobs, total_jobs)
+        else:
+            return '-/-'
+
+    def _get_data_from_row(row, data, headers):
```

Comment:
Could we maybe remove `data` as parameter, and return a new list every time? So down, in `workflow_status` scope, when we call it we are always in control of what goes inside `data` making it easier to debug?